### ISSUE TITLE
New-GPWMIFilter has a hardcoded namespace

### ISSUE THREAD

**Issue by djpbessems** [FriedrichWeinmann/GPWmiFilter#4]
Currently, line [118](https://github.com/FriedrichWeinmann/GPWmiFilter/blob/833b1536487365b4049dc92a162187c36977e92f/GPWmiFilter/functions/New-GPWmiFilter.ps1#L118) of `New-GPWmiFilter.ps1` has a hardcoded namespace of `root\CIMv2`.

Can this module be updated to allow WMI Filters in other namespaces (I would like to use `root\rsop\computer` for instance)?

**Comment by FriedrichWeinmann** [comment]
Sounds reasonable.
Implemented in v1.0.3 now available on the PSGallery.

Both New-GPWmiFilter and Set-GPWmiFilter now have a `-Namespace` parameter that defaults to `root\CIMv2`.
Furthermore, you can now include filter conditions from multiple namespaces by overriding that namespace in individual filter expressions by prefixing the namespace:
`-Filter 'root\rsop\computer;<filter>'` and `-Namespace 'root\rsop\computer' -Filter '<filter>'` are now equivalent

**Comment by djpbessems** [comment]
I just tested the new version, but I'm seeing unexpected results (looks like there's a string being split on the wrong lengths?):
![image](https://user-images.githubusercontent.com/23643312/127138080-d6d996d7-ae3e-48b2-9180-e30710a452ba.png)
![image](https://user-images.githubusercontent.com/23643312/127138095-e0f24200-5e75-4456-9550-00fd668ddbd8.png)

The output of adding the filters in powershell looks like this (and I'm not getting an error):
```
Filter
------
1;3;10;55;WQL;;SELECT * FROM Win32_ComputerSystem Where Dom...
1;3;10;74;root\RSOP\Computer;SELECT * FROM RSOP_Session...
```

**Comment by FriedrichWeinmann** [comment]
Should be fixed now in 1.0.4
Forgot to update the namespace length calculation ... and tested it against one that happened to have the same length :(

**Comment by djpbessems** [comment]
Tested 1.0.4 and everything seems to work correctly!
